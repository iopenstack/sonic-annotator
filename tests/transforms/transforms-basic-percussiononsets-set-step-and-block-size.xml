<transform
    id="vamp:vamp-example-plugins:percussiononsets:onsets"
    stepSize="345"
    blockSize="512"
   />
