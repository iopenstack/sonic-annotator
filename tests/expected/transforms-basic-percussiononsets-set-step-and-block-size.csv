"./audio/3clicks8.wav",0.782312925
,1.650680272
,2.417346939
